ethcore/wasm/src/runtime.rs

```diff
@@ -211,6 +249,9 @@ impl<'a, 'b> Runtime<'a, 'b> {
 
 		let code = self.memory.get(code_ptr, code_len as usize)?;
 
+		charge!(self, self.ext.schedule().create_gas as u32);
+		charge!(self, (self.ext.schedule().create_data_gas * code.len()) as u32);
```

Comment:
again, using u32 for gas here seems a little flaky. In the EVM code we use u64 in general.